src/framework/ui/index.ts

```diff
@@ -7,14 +7,18 @@ import {
   RadioGroup as RadioGroupComponent,
   Props as RadioGroupProps,
 } from './radioGroup/radioGroup.component';
+import {
+  Toggle,
```

Comment:
The same as in [Layout](https://github.com/akveo/react-native-ui-kitten/pull/240#discussion_r249254909)